config/core/test-configs.yaml

```diff
@@ -2025,6 +2025,13 @@ test_configs:
   - device_type: imx6q-sabrelite
     test_plans:
       - baseline
+      - kselftest-filesystems
+      - kselftest-futex
+      - kselftest-lib
+      - kselftest-livepatch
```

Comment:
Tested OK:
https://staging.kernelci.org/test/plan/id/623378250a513edc394b312e/
However, most of the livepatch test cases were skipped.